gulpfile.js

```diff
@@ -27,7 +27,7 @@ gulp.task('serve', gulp.series('build', function(done){
 gulp.task('watch', function() {
   gulp.watch('docs/**/*', gulp.series('docs', browserReloadSync));
   gulp.watch(['docs/layout/*.html', 'docs/partials/*{html,hbs}', 'docs/assets/partials/*{html,hbs}', 'node_modules/foundation-docs/templates/*{html,hbs}'], gulp.series('docs:all', browserReloadSync));
-  gulp.watch('scss/**/*', gulp.series('sass:foundation', browserReloadSync));
+  gulp.watch('scss/**/*', gulp.series('sass', browserReloadSync));
```

Comment:
Looks good to me!
https://github.com/zurb/foundation-sites/blob/f74c6f7dba5e9c4c57e48318d4e1631ede4f6f90/gulp/tasks/sass.js#L17